.github/workflows/ubuntu.yml

```diff
@@ -164,6 +208,7 @@ jobs:
           ls -la "${RNP_INSTALL}/lib/"librnp*.so
           ! ctest -N | grep 'ruby' > /dev/null
           ! ls -la src/tests/ | grep 'ruby' > /dev/null
+
```

Comment:
Do we need this additional empty line?

Reply:
No, this additional empty line is optional. I will remove it. Thank you.